### DISCUSSION TITLE
rights problem

### DISCUSSION THREAD

**Discussion by cultd3ad** [stuttter/ludicrousdb#170] [General]
Hello,
the following problem, as soon as the db.php is installed in the /wp-content/ folder, I can’t not create any pages or posts. Not all content such as categories are displayed for selection, see the screen. The following error is in the log file.

I delete the db.php, everything works fine, can someone help me ??
thanks in advance

Wordpress Version: 6.0.3
PHP Version: 7.4.3
MySQL Version: 5.7 and 8.0 same issue.

Link to Screen
http://www.bilderload.com/daten/reviewGPQZ7.png

when i clik to save the post, i become this error:
` Got error 'PHP message: WordPress database error Expression #1 of ORDER BY clause is not in SELECT list, references column 'sport.t.name' which is not in SELECT list; this is incompatible with DISTINCT for query \n\t\t\tSELECT DISTINCT t.term_id, tr.object_id\n\t\t\tFROM wpoa_terms AS t  INNER JOIN wpoa_term_taxonomy AS tt ON t.term_id = tt.term_id INNER JOIN wpoa_term_relationships AS tr ON tr.term_taxonomy_id = tt.term_taxonomy_id\n\t\t\tWHERE tt.taxonomy IN ('mt-listing-category2', 'mt-listing-tags', 'mt-listing-amenities', 'mt-listing-languages', 'mt-listing-currencies', 'mt-listing-payment-methods', 'mt-bookmaker-restricted') AND tr.object_id IN (52831, 53120, 61990, 53839, 53984, 53432, 53974, 53712, 53612, 54181, 62303, 62299, 69965, 69959, 69955, 69944, 69933, 69927, 69917, 69908, 69901, 69891, 69860, 69859, 69842, 69837, 69828, 69766, 69762, 69758, 69754, 69750, 69740, 69736, 69732, 69728, 69724, 69715, 69711, 69707, 69703, 69699, 69695, 69690, 69685, 69681, 69668, 69664, 69660, 69656, 69652, 69643, 69639, 69635, 69631...PHP message: WordPress database error Expression #1 of ORDER BY clause is not in SELECT list, references column 'sport.t.name' which is not in SELECT list; this is incompatible with DISTINCT for query \n\t\t\tSELECT DISTINCT t.term_id, tr.object_id\n\t\t\tFROM wpoa_terms AS t  INNER JOIN wpoa_term_taxonomy AS tt ON t.term_id = tt.term_id INNER JOIN wpoa_term_relationships AS tr ON tr.term_taxonomy_id = tt.term_taxonomy_id\n\t\t\tWHERE tt.taxonomy IN ('mt-slot-category') AND tr.object_id IN (73425, 73424, 73423, 73422, 73421, 73420, 73426, 215393, 215357, 133522, 133486, 133467, 133461, 133453, 133451)\n\t\t\tORDER BY t.name ASC\n\t\t\t\n\t\t made by WP_Term_Query::get_termsPHP message: WordPress database error Expression #1 of ORDER BY clause is not in SELECT list, references column 'sport.t.name' which is not in SELECT list; this is incompatible with DISTINCT for query \n\t\t\tSELECT DISTINCT t.term_id, tr.object_id\n\t\t\tFROM wpoa_terms AS t  INNER JOIN wpoa_term_taxonomy AS tt ON t.term_id = tt.term_id INNER JOIN wpoa_term_relationships AS tr ON tr.term_taxonomy_id = tt.term_taxonomy_id\n\t\t\tWHERE tt.taxonomy IN ('mt-bonus-category') AND tr.object_id IN (61992, 69960, 69956, 69945, 69934, 69929, 69918, 69910, 69902, 69892, 69869, 69862, 69843, 69838, 69829, 69767, 69763, 69759, 69755, 69751, 69741, 69737, 69733, 69729, 69725, 69716, 69712, 69708, 69704, 69700, 69696, 69691, 69686, 69682, 69669, 69665, 69661, 69657, 69654, 69644, 69641, 69637, 69633, 69628, 69619, 69614, 69609, 69604, 69598, 69580, 69569, 69555, 69543, 69529, 69497, 66352, 66349, 66223, 66216, 66205, 66110, 66105, 66094, 66088, 66077, 65414, 64688, 64666, 64659, 64653, 64064, 64037, 64034, 64028, 64019, 63982, 639...PHP message: WordPress database error Expression #1 of ORDER BY clause is not in SELECT list, references column 'sport.t.name' which is not in SELECT list; this is incompatible with DISTINCT for query \n\t\t\tSELECT DISTINCT t.term_id, tr.object_id\n\t\t\tFROM wpoa_terms AS t  INNER JOIN wpoa_term_taxonomy AS tt ON t.term_id = tt.term_id INNER JOIN wpoa_term_relationships AS tr ON tr.term_taxonomy_id = tt.term_taxonomy_id\n\t\t\tWHERE tt.taxonomy IN ('mt-knowledge-category') AND tr.object_id IN (20404, 20406, 20407, 20408, 48400)\n\t\t\tORDER BY t.name ASC\n\t\t\t\n\t\t made by WP_Term_Query::get_terms', referer: https://*fa.de/wp-admin/post-new.php`

**Comment by RichardTMiles** [comment]
This is a general mysql error
